api/api/error_handler.py

```diff
@@ -55,6 +55,42 @@ def _cleanup_detail_field(detail: str) -> str:
     return ' '.join(str(detail).replace("\n\n", ". ").replace("\n", "").split())
 
 
+async def handle_expect_header(request: ConnexionRequest, exc: ContentSizeExceeded = None) -> ConnexionResponse:
+    """Handler for 'Expect' header.
+    
+    Parameters
+    ----------
+    request : ConnexionRequest
+        Incoming request.
+    exc : ContentSizeExceeded
+        Raised exception if content size exceeds the limit.
+
+    Returns
+    -------
+    Response
+        HTTP Response returned to the client.
+    """
+    problem = {
+       "title": "error : 417",
+       "detail": "Expectation Failed."
+    }
```

Comment:
I think this should be

```suggestion
    problem = {
       "title": "Expectation failed",
       "detail": "Unknown Expect",
       "error": 417
    }
```

but we should use `WazuhError` like in the rest of the code instead of hardcoding it. I'm not sure if raising an error here will be catched though.